packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/changeListMorph.st

```diff
@@ -0,0 +1,4 @@
+accessing
+changeListMorph
+
+	^ self browserWindow submorphs detect: [:a | a externalName = 'change list']
```

Comment:
Inside `buildSingleSelectionChangeListSpec:`.
Maybe we should put the string into a method on the class side so you can easily search for sender's instead of relying on a magic string in both places?

Reply:
Sounds like a good idea